src/lib/encoder.ts

```diff
@@ -44,14 +44,11 @@ import {
   toBN,
 } from 'web3-utils';
 
-import { JSONURLDataToEncode, URLDataWithHash } from '../types';
+import { JSONURLDataToEncode, URLDataWithHash, Verification } from '../types';
```

Comment:
What is this type? Not clear when reading it

Reply:
It's the JSON version of the JSONURL, Hmmm, this is not new though.